dist/windows/installer-translations/portugueseBR.nsi

```diff
@@ -15,7 +15,7 @@ LangString inst_magnet ${LANG_PORTUGUESEBR} "Abrir links magnéticos com qBittor
 ;LangString inst_firewall ${LANG_ENGLISH} "Add Windows Firewall rule"
 LangString inst_firewall ${LANG_PORTUGUESEBR} "Adicionar regra no firewall do Windows"
 ;LangString inst_pathlimit ${LANG_ENGLISH} "Disable Windows path length limit (260 character MAX_PATH limitation, requires Windows 10 1607 or later)"
-LangString inst_pathlimit ${LANG_PORTUGUESEBR} "Disable Windows path length limit (260 character MAX_PATH limitation, requires Windows 10 1607 or later)"
+LangString inst_pathlimit ${LANG_PORTUGUESEBR} "Desativar limite de caracteres na variável path do Windows (limite de 260 caracteres, requer Windows 10 1067 ou melhor)"
```

Comment:
Wouldn't one of these two better convey the meaning?

"1607 ou melhor" -> "1607 ou mais recente"
or
"1607 ou melhor" -> "1607 ou superior"

Also, the original text does not explicitly mention the `PATH`environment variable, so I think a better translation would be something along the lines of "Desativar limitação/limite de comprimento de caminhos do Windows". I'm a native pt-PT speaker, so if this (or some close alternative) dos not sound idiomatic in pt-BR, you can keep it as it is. But please capitalize `path` in that case, since you are actually referring to the `PATH` environment variable.

Reply:
I didn't translate the word 'path' because there's no translation - from what I could gather online and from the lack of translation on Windows itself - and didn't capitalize it because it was not originally capitalized. 
Oh... You're right. It doesn't refer to the PATH environment variable. Okay, yeah, you're right.